id-repository/id-repository-core/src/main/java/io/mosip/idrepository/core/manager/CredentialServiceManager.java

```diff
@@ -136,7 +136,7 @@ public class CredentialServiceManager {
 	@Autowired
 	private PartnerServiceManager partnerServiceManager;
 
-	@Autowired
+	@Autowired(required = false)
 	private HandleRepo handleRepo;
```

Comment:
Since `handleRepo` can be null, handle null where this field is used.

Reply:
Done